docs/api-guide/authentication.md

```diff
@@ -367,6 +367,10 @@ The following example will authenticate any incoming request as the user given b
 
 The following third-party packages are also available.
 
+## DRF Auth Kit
+
+[DRF Auth Kit][drf-auth-kit] library provides a modern REST authentication solution with JWT cookies, social login, multi-factor authentication, and comprehensive user management. The package offers full type safety, automatic OpenAPI schema generation with DRF Spectacular. It supports multiple authentication types (JWT, DRF Token, or Custom) and includes built-in internationalization for 50+ languages.
```

Comment:
Don't take this the wrong way, but could this new-ish kid on the block be placed a bit further down in the page. I don't think it's fair to place it above other well established packages like `django-rest-knox`.